Applications/Spire/Source/TimeAndSales/TimeAndSalesTableView.cpp

```diff
@@ -36,10 +36,7 @@ namespace {
       set(BackgroundColor(Qt::transparent));
     style.get(body_selector > CurrentColumn()).
       set(BackgroundColor(Qt::transparent));
-  }
-
-  auto apply_table_header_style(StyleSheet& style) {
-    style.get(Any()).
+    style.get(Any() > (is_a<TableHeader>() < is_a<Box>())).
```

Comment:
Can you explain what this is trying to do? Why can't you style the ```TableView``` directly?

Reply:
I want to add a border line to the `TableHeader`. I don't think it can be achieved by styling the `TableView` directly, since there is no `Box` inside the `TableHeader` to style a border, and there is no style selector for the border in the `TableView.
![image](https://github.com/spiretrading/nexus/assets/75099914/a0579431-b718-4858-964d-8a57cabb6e51)

Reply:
If it's not in the spec then don't add it.

Reply:
Okay. It's not in the spec, but in the updated specs of the table header. I will remove it.

Reply:
If it's in the table header spec then don't implement it in the time and sales. Doing this spreads out logic all of the codebase and makes it very hard to follow.

Things related to the table header should be implemented in TableHeader or TableHeaderItem, or whatever suitable abstraction is directly related to that functionality, as opposed to taking things like the table and smearing it's implementation everywhere.